functests/performance/cpu_management.go

```diff
@@ -146,23 +147,32 @@ func execCommandOnWorker(cmd []string, workerRTNode *corev1.Node) string {
 	return strings.Trim(string(out), "\n")
 }
 
-func CPUListConverter(s string) ([]string, error) {
-	results := []string{}
-	for _, cpuRange := range strings.Split(string(s), ",") {
-		if strings.Contains(cpuRange, "-") {
-			seq := strings.Split(cpuRange, "-")
-			if len(seq) != 2 {
-				return nil, fmt.Errorf("incorrect CPU range: %q", cpuRange)
+func CPUListConverter(cpusetLine string) (cpusList []string, err error) {
```

Comment:
Can you change this method https://github.com/openshift-kni/performance-addon-operators/blob/d6d9ca9854b37d182491472d4e15dbd23126f4dd/pkg/controller/performanceprofile/components/utils.go#L35 to be imported and re-use it?